exe/wallet/http-bridge/Main.hs

```diff
@@ -277,9 +277,11 @@ cmdServe = command "serve" $ info (helper <*> cmd) $ mempty
             -> DBLayer IO s t
             -> IO (WalletLayer s t)
         newWalletLayer (sb, tracer) db = do
-            (nl, block0, feePolicy) <- newNetworkLayer (sb, tracer)
+            (nl, block0, theFeePolicy) <- newNetworkLayer (sb, tracer)
```

Comment:
Minor style note but: `the` makes the variable name longer but doesn't buy us much in terms of clarity. If there's a name-clash with `BlockchainParameters`, I'd suggest to prefix `BlockchainParameters` with `get` or `_` but leave variable name bindings simple :+1:

Reply:
done